src/main/kotlin/com/openlattice/chronicle/services/upload/AppDataUploadService.kt

```diff
@@ -41,19 +41,19 @@ import java.util.function.Consumer
  */
 
 class AppDataUploadService(
-    private val storageResolver: StorageResolver,
-    private val scheduledTasksManager: ScheduledTasksManager,
-    private val enrollmentManager: EnrollmentManager,
-    private val organizationSettingsManager: OrganizationSettingsManager
+        private val storageResolver: StorageResolver,
+        private val scheduledTasksManager: ScheduledTasksManager,
+        private val enrollmentManager: EnrollmentManager,
+        private val organizationSettingsManager: OrganizationSettingsManager
 ) : AppDataUploadManager {
```

Comment:
Consider merging with current upload service, but might be easier to keep separate.

Reply:
I think it's easier to keep separate